oxd-server/src/test/resources/oxd-server-jenkins.yml

```diff
@@ -21,14 +21,15 @@ add_client_credentials_grant_type_automatically_during_client_registration: true
 migration_source_folder_path: ''
 protect_commands_with_oxd_id: []
 allowed_op_hosts: []
-storage: h2
+storage: gluu_server_configuration
 enable_tracing: false
 # tracer: jaeger or zipkin
 tracer: 'jaeger'
 tracer_host: 'localhost'
 tracer_port: 5775
 storage_configuration:
-  dbFileLocation: "./oxd_db"
+  location: 'D:/oxd-repo'
```

Comment:
We don't have this on jenkins machine, do we?

Reply:
yes it's by mistake. I have corrected.